modules/ROOT/pages/to-publish-assets-maven.adoc

```diff
@@ -214,7 +219,7 @@ This value tells the platform that you're using a token.
 
 == Dependencies external to Exchange
 
-Java libraries and POM files cannot currently be published as Exchange assets.
+Java libraries and POM files can be published as Exchange custom assets.
 
 Mule 4 Extensions that declare dependencies that don't exist in Maven Central or the MuleSoft Maven repositories are not currently supported.
 
```

Comment:
I can not select this line
```
When you use Maven Facade to publish connectors, examples and templates, a documentation portal is automatically created for them in Exchange and you can xref:to-describe-an-asset.adoc[document them normally].
```

but let's change it to:

```
When you use Maven Facade to publish connectors, policies, customs, examples and templates, a documentation portal is automatically created for them in Exchange and you can xref:to-describe-an-asset.adoc[document them normally].
```

Reply:
Good point. Done.